src/components/Buttons.js

```diff
@@ -2,17 +2,18 @@ import React from 'react';
 import './styles.css';
 import PropTypes from 'prop-types';
 
-const Buttons = ({ number }) => {
+const Buttons = ({ number, handleEvent }) => {
   if (number === '\u00F7' || number === 'x' || number === '-' || number === '+' || number === '=') {
-    return <div className="btn orange">{number}</div>;
+    return <div onClick={handleEvent} onKeyDown={handleEvent} role="presentation" className="btn orange">{number}</div>;
   } if (number === '0') {
-    return <div className="btn zero">{number}</div>;
+    return <div onClick={handleEvent} onKeyDown={handleEvent} role="presentation" className="btn zero">{number}</div>;
   }
-  return <div className="btn">{number}</div>;
+  return <div onClick={handleEvent} onKeyDown={handleEvent} role="presentation" className="btn">{number}</div>;
 };
 
 Buttons.propTypes = {
   number: PropTypes.number.isRequired,
+  handleEvent: PropTypes.func.isRequired,
 };
 
 export default Buttons;
```

Comment:
- [ ]  (Optional) Your app works fine but consider fixing the following console error. 

`Warning: Failed prop type: Invalid prop number of type string supplied to Buttons, expected number.`

This error is indicating that you are passing a prop named number to a component called Buttons, but the expected type for this prop is a number, and you're passing a string instead. To fix this, you should ensure that when you use the Buttons component, you provide a prop with a value of type number.

e.g

```
<Buttons number={42} />
```